### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by justinrlle**
Summary of the edits:
- clarified a few things in the project presentation
- slightly clarified instructions on contributing
- more detailed installation instructions
- a new section for the frontend
- cleaned up the links at the end generated by `mix phx.new`

Questions to resolve:
- ⚠️ clarification about user registration
- db setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,36 +2,61 @@
...
+$ mix local.hex
+
+# install phoenix
+$ mix archive.install https://github.com/phoenixframework/archives/raw/master/phx_new.ez
+```
+If it fails, please have a look at <https://hexdocs.pm/phoenix/installation.html>
+
+For [`postgresql`], install using your prefered way, and setup the DB to match the config in [`config/dev.exs`](./config/dev.exs).
+
+> :warning: This parts needs some serious changes, because right now, the config for postgresql is right in the repository, checked in, absolutly not flexible. We're still looking for a standard solution, matching what's being done in the real world (`env` variables, for exemple), but which still looks fine in regard to the best practices in the `erlang/OTP` world.
```
Comment (justinrlle) [inline README.md]: The [RealWorld app for elixir](https://github.com/gothinkster/elixir-phoenix-realworld-example-app) use environment variables for prod environment (eg `System.get_env("DATABASE_URL")`, and provides no `config/dev.exs`, but a template, that you must copy and then edit to your need.
The standard for nearly any other kind of app is [`dotenv`](https://www.google.com/search?hl=en&q=dotenv), but it looks like it [is'nt the elixir way](https://github.com/avdi/dotenv_elixir), and so should be avoided.
So I think we should be using something like the `config/prod.secret.exs`, but for `dev`, which won't be checked in.
Anyway, that's a problem that needs to be solved.

**Comment by justinrlle** [comment]
@Shenrak, can you quickly review this?

File: README.md [hunk 2]
```diff
@@ -1,37 +1,62 @@
...
+This project aims to provide a REST API for a RPG-oriented chat. The API will be exposed through an elixir backend writen using the Phoenix framework, and will be open-source and self-hostable. We want to offer an alternative to people that would like to get in the P&P RPG world but who do not feel confident about speaking with strangers or putting themselves forward.
 
-The game master will have features to easely handle the means of a P&P RPG. We also want to provide a global hub in which you can join a session whenever the game master is able to introduce you to the game.
+The game master will have features to easely handle the means of a P&P RPG (features yet to be fully defined). We also want to provide a global hub in which you can join a session whenever the game master is able to introduce you to the game. The list of features is still fluctuating, expect new things here!
 
-For the first version, we do not want to force users to create accounts.
+For the first version, we do not want to force users to create accounts, kind of like IRC. TODO: but will it be available?
```
Comment (justinrlle) [inline README.md]: Oh yeah, I'm not sure about that part, will the user registration be optional, or not implemented?

Comment (Shenrak) [reply README.md]: For now we can't say, it's in the thinking pipe ¯\_(ツ)_/¯

File: README.md (hunk 1 above)
Comment (Shenrak) [reply README.md]: I understand what you're saying here, feel free to create another pull request to make that change, as i see no problem with it

**Comment by justinrlle** [comment]
I'll clean the config in another commit, the warning in the README will fulfill its role in the meantime.

You can go and merge this

**Pull request merged**